CHANGELOG.md

```diff
@@ -42,6 +42,10 @@ Bottom level categories:
 
 ### Bug Fixes
 
+#### General
+
+- Derive storage bindings via `naga::StorageAccess` instead of `naga::GlobalUse`. By @teoxoy in [#3985](https://github.com/gfx-rs/wgpu/pull/3985).
```

Comment:
nitpick: Might it be worth mentioning what kinds of validation errors would now be unblocked for users?

Reply:
I think it's quite niche the situation you'd have to be in where you'd get a validation error from dx12. You'd have to use an `auto` layout, declare the storage buffer in the shader with `read_write` access but only `read` from it and run on dx12 (which is not the default).